benches/storage/scan.rs

```diff
@@ -61,31 +55,171 @@ fn bench_tombstone_scan(b: &mut Bencher) {
                 k.clone(),
                 ts,
             )
-            .expect("");
+            .unwrap();
         store
             .commit(
                 Context::new(),
                 vec![Key::from_raw(&k)],
                 ts,
                 ts_generator.next().unwrap(),
             )
-            .expect("");
+            .unwrap();
     }
 
+    // We should got nothing when scanning any keys.
+
     kvs = KvGenerator::new(100, 1000);
+    let ts = ts_generator.next().unwrap();
     b.iter(|| {
         let (k, _) = kvs.next().unwrap();
         assert!(
             store
                 .scan(
-                    Context::new(),
+                    super::util::new_no_cache_context(),
                     Key::from_raw(&k),
                     1,
                     false,
-                    ts_generator.next().unwrap()
+                    ts
                 )
                 .unwrap()
                 .is_empty()
         )
     })
 }
+
+fn bench_scan(b: &mut Bencher, forward_scan: bool, versions: usize, long_value: bool) {
+    const NUMBER_KEYS: usize = 10000;
```

Comment:
Please make number_keys configurable.

Reply:
Seems all key/value pairs are in memtable?

Reply:
Seems that we need dynamic benches for different kind of keys and values.

Reply:
Benchmarking 100000 keys are really slow.. even 6 hours is not enough for completing all benchmarks. Do we really have the necessity to test the difference of memtable and non-memtable access? Seems that any kind of optimizations by us won't make a difference over this.